tao_compiler/mlir/disc/transforms/disc_to_llvm.cc

```diff
@@ -414,6 +412,41 @@ Value ConvertLaunchFuncOpToRalCallPattern::generateParamsArray(
   auto arguments = getTypeConverter()->promoteOperands(
       loc, launch_op.getOperands().take_back(num_kernel_operands),
       operands.take_back(num_kernel_operands), builder);
+  bool mem_intensive_opt_experimental = false;
+  tensorflow::ReadBoolFromEnvVar("DISC_MEM_INTENSIVE_OPT_EXPERIMENTAL", false,
```

Comment:
please only init such env in one place. An example is here: https://github.com/alibaba/BladeDISC/blob/main/tao_compiler/mlir/disc/disc_util.cc#L196

Reply:
Fixed. Thanks.